packages/@react-aria/aria-modal-polyfill/docs/watchModals.mdx

```diff
@@ -17,6 +17,7 @@ import packageData from '@react-aria/aria-modal-polyfill/package.json';
 ---
 category: Overlays
 keywords: [dialog, modal, aria]
+hidden: true
```

Comment:
This one is preserved but just hidden from the sidebar in the docs. The new overlays system won't require this to be used since it hides all elements outside the provider by default. But as of this PR we haven't yet switched to the new system in RSP, so this is still linked to from the RSP dialog docs.

Reply:
Can we remove this page now that we've switched over to the new overlay hooks?